Dockerfile

```diff
@@ -24,3 +28,10 @@ CMD ./bin/web
 
 FROM hyku-web AS hyku-worker
 CMD ./bin/worker
+
+FROM solr:8.3 AS hyku-solr
```

Comment:
Added here for tests and to be able to pull the initialize_app service locally to override the one we extend (inherit) from/to hyku's docker compose file.